src/instruments/src/SD/Pages/Hyd/Hyd.tsx

```diff
@@ -0,0 +1,473 @@
+                    pressure={greenPressure}
+                    hydLevel={greenHydLevel}
+                    x={110}
+                    y={y}
+                    fireValve={greenFireValve}
+                    pumpPBStatus={greenPumpPBStatus}
+                    yellowElectricPumpStatus={yellowElectricPumpStatus}
+                />
+                <HydSys
+                    title="BLUE"
+                    pressure={bluePressure}
+                    hydLevel={blueHydLevel}
+                    x={300}
+                    y={y}
+                    fireValve={0}
+                    pumpPBStatus={bluePumpPBStatus}
+                    yellowElectricPumpStatus={yellowElectricPumpStatus}
+                />
+                <HydSys
+                    title="YELLOW"
+                    pressure={yellowPressure}
+                    hydLevel={yellowHydLevel}
+                    x={490}
+                    y={y}
+                    fireValve={yellowFireValve}
+                    pumpPBStatus={yellowPumpPBStatus}
+                    yellowElectricPumpStatus={yellowElectricPumpStatus}
+                />
+
+                <PTU x={300} y={y + 126} ptuScenario={ptuScenario} />
+
+                <RAT x={290} y={y} />
+
+                <text
+                    id="ELEC-centre"
+                    className={!ACBus1IsPowered ? 'RatPtuElec FillAmber' : 'RatPtuElec FillWhite'}
+                    x={350}
+                    y={y + 245}
+                    alignmentBaseline="central"
+                >
+                    ELEC
+
+                </text>
+
+                <text
+                    id="ELEC-right"
+                    className={!ACBus2IsPowered ? 'RatPtuElec FillAmber' : 'RatPtuElec FillWhite'}
+                    x={548}
+                    y={y + 180}
+                    alignmentBaseline="central"
+                >
+                    ELEC
+
+                </text>
+                <Triangle x={500} y={y + 180} colour={elecTriangleColour} fill={elecTriangleFill} orientation={-90} />
+                <line className={elecRightFormat} x1={490} y1={y + 180} x2={500} y2={y + 180} />
+
+                <text className="Psi" x={205} y={y + 70} alignmentBaseline="central">PSI</text>
+                <text className="Psi" x={395} y={y + 70} alignmentBaseline="central">PSI</text>
+
+            </svg>
+        </>
+    );
+};
+
+type RATProps = {
+    x: number,
+    y: number,
+}
+
+const RAT = ({ x, y }: RATProps) => {
+    const [RatStowed] = useSimVar('L:A32NX_HYD_RAT_STOW_POSITION', 'percentover100', 500);
+
+    return (
+        <>
+            <text className="RatPtuElec FillWhite" x={x - 42} y={y + 180} alignmentBaseline="central">RAT</text>
+            <line className={`GreenLine ${RatStowed === 1 ? '' : 'Hide'}`} x1={x} y1={y + 180} x2={x + 10} y2={y + 180} />
+            <Triangle x={x} y={y + 180} colour={RatStowed === 1 ? 'Green' : 'White'} fill={RatStowed === 1 ? 1 : 0} orientation={90} />
```

Comment:
0 to 1, 0 when stowed indeed. Not entirely sure what means ===, is it a sctrict equal?
I'm not entirely sure if there is a deployed sensor, nor if that sensor would be giving ecam the rat status when just deploying or in fully extended position.  
Still I suggest whatever you chose you have some margin like a position sensor would have. So something like > 0.1 or > 0.9, because this position might at some point reflect a physical object subject to aerodynamic forces that could oscillate a bit for example being always between 0.99 and 1.

Reply:
@lukecologne You might be right, although the changeover valve also needs to be receiving an adequate voltage...whatever that means.

Reply:
Deploying electrical power is handled in #4810. Rat model is still placeholder though, so any check on position different from 0 is good for the moment anyway. When giving some love to RAT, we might end up with a simple boolean var simpler to handle on your side. I'll need more technical info for this :)